pew/pew.py

```diff
@@ -188,7 +190,9 @@ def shell(env, cwd=None):
         if 'CMDER_ROOT' in os.environ:
             shell = 'Cmder'
         elif windows:
-            shell = 'powershell'
+            c = delegator.run('tasklist /fi "PID eq {0}" /fo csv /nh'.format(os.getppid()))
```

Comment:
There's no reason to use delegator instead of subprocess here, right?

Also, I know that this string interpolation is safe, but I try to avoid having such interpolations at all in risky contexts like this one (I wouldn't want people to look at this code as an example, forget about potential issues and then shooting themselves in the foot... we ought to steward good practices :) )

Reply:
No, using delegator was just a personal choice as it's written in pure python and is just essentially a nice wrapper around subprocess. 

How would you recommend avoiding the use of a string interpolation?

Thanks for the feedback.

Reply:
Are you ok with the use of delegator? Or would you rather I use subprocess?